alacritty_terminal/src/term/mod.rs

```diff
@@ -745,6 +745,12 @@ pub struct Term<T> {
 
     /// Current forward and backward buffer search regexes.
     regex_search: Option<RegexSearch>,
+
+    /// Width of individual cell.
+    cell_width: f32,
+
+    /// Height of individual cell.
+    cell_height: f32,
```

Comment:
What's the reason for saving the cell width/height here, when what we actually need is the text area size? Wouldn't it be easier to just store the only thing we need?

If we store more information than necessary, why wouldn't we just store `SizeInfo`?

Reply:
Initially I didn't want to copy the `SizeInfo` to reduce the final size of `Term`, but I guess that the difference is too small to be worth it. In 5581bf7a6ca31faaf2e41402764cbe9b9365c42b it is changed to keep a copy of `SizeInfo`.